docs/spec/typeddict.rst

```diff
@@ -27,13 +27,13 @@ supported by ``typing.NamedTuple``.  Other features include
 TypedDict inheritance and totality (specifying whether keys are
 required or not).
 
-This section also provides a sketch of how a type checker is expected
-to support type checking operations involving TypedDict objects.
-Similar to :pep:`484`, this discussion is left somewhat vague on purpose,
-to allow experimentation with a wide variety of different type
-checking approaches.  In particular, type compatibility should be
-based on structural compatibility: a more specific TypedDict type can
-be compatible with a smaller (more general) TypedDict type.
+This section also provides a sketch of how a type checker is expected to
+support type checking operations involving TypedDict objects. Similar to
+:pep:`484`, this discussion is left somewhat vague on purpose, to allow
+experimentation with a wide variety of different type checking approaches.  In
+particular, :term:`assignability <assignable>` should be structural: a more
+specific TypedDict type can be assignable to a smaller (more general)
+TypedDict type.
```

Comment:
Could perhaps clarify slightly what "structural assignability" means here:

```suggestion
TypedDict type even if the larger one is not an explicit subclass of the
smaller one.
```

Should "structural assignability" be added as a glossary term? You use it in the protocols doc as well